### PR TITLE
Unify private key format that is used in interfaces in WIF 

### LINKED ISSUE #120: Unify private key format as WIF in places where treats private key
The current situation is the cause of confusion for users who want to start up a new network.

* `generatetoaddress` RPC accepts a hex private key, but `tapyrus-genesis` command accepts a WIF private key.
* The help document of `tapyrus-genesis` command doesn't show details of `-signblockprivatekey`. Actually, the description is placed in the code but it is set as a hidden option.
* Other places where are before forked uses WIF format. but they don't show to use WIF explicitly.

So, I think we should do below.

* Unify the format of private keys as WIF.
* Make the `-signblockprivatekey` help document is shown.
* Make `importprivkey` RPC and `tapyrus-tx` command show to use WIF explicitly.

### PR DESCRIPTION
**Pull request by rantan**
This PR fixes #120. see it before review.

### CODE REVIEW

File: src/rpc/mining.cpp [hunk 1]
```diff
@@ -115,10 +115,7 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
     std::shared_ptr<CReserveScript> coinbaseScript = std::make_shared<CReserveScript>();
     coinbaseScript->reserveScript = GetScriptForDestination(destination, colorId);
 
-    std::vector<unsigned char> privkeyraw = ParseHex(request.params[2].get_str());
-    CKey cPrivKey;
-    cPrivKey.Set(privkeyraw.begin(), privkeyraw.end(), true);
-
+    CKey cPrivKey = DecodeSecret(request.params[2].get_str());
     if(!cPrivKey.size())
```
Comment (azuchi) [inline src/rpc/mining.cpp:119]: It is more intuitive to use `cPrivKey.IsValid()` to check the validity of the private key. In addition, it should be checked if the private key corresponds to `FederationParams().GetLatestAggregatePubkey()`.

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -3952,21 +3952,18 @@ UniValue generate(const JSONRPCRequest& request)
...
     }
 
     int num_generate = request.params[0].get_int();
 
-    std::vector<unsigned char> privkeyraw = ParseHex(request.params[1].get_str());
-    CKey cPrivKey;
-    cPrivKey.Set(privkeyraw.begin(), privkeyraw.end(), true);
-
+    CKey cPrivKey = DecodeSecret(request.params[1].get_str());
     if(!cPrivKey.size())
```
Comment (azuchi) [inline src/wallet/rpcwallet.cpp:3967]: It is more intuitive to use `cPrivKey.IsValid()` to check the validity of the private key. In addition, it should be checked if the private key corresponds to `FederationParams().GetLatestAggregatePubkey()`.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: src/rpc/mining.cpp (hunk 1 above)
Comment (rantan) [reply src/rpc/mining.cpp]: fixed at 37cd8f627f64269901a0954934ecaa7e614a6407

File: src/wallet/rpcwallet.cpp (hunk 2 above)
Comment (rantan) [reply src/wallet/rpcwallet.cpp]: fixed at 37cd8f627f64269901a0954934ecaa7e614a6407

**Review by azuchi — APPROVED** [review]

**Pull request merged**